### DISCUSSION TITLE
pipx install ignores packages installed by OS ?

### DISCUSSION THREAD

**Discussion by rumly111** [pypa/pipx#1394] [Q&A]
I installed a package that had `lxml` as it's dependency. In my OS (ubuntu) I had installed python3-lxml , and it was the version that satisfied the requirement of the package I wanted to install via pipx . But pipx still installed lxml in the program's venv, ignoring global python library `lxml` . Why did it happen, and how to prevent it?

**Comment by huxuan** [comment]
"pipx install --system-site-packages" should be what you want. For more information, please refer to `pipx install --help`

**Reply by rumly111** [reply]
I thought so as well, but --system-site-packages didn't help, maybe because python3-lxml installs into /usr/lib/python3/dist-packages

**Reply by huxuan** [reply]
I did not test it, but what I know is that the `--system-site-packages` argument is passed directly to the `venv` module, so even if it is true, there does not seem to be anything `pipx` can do.
